bentoml/_internal/runner/runner_handle/remote.py

```diff
@@ -100,11 +99,9 @@ def strip_query_params(url: yarl.URL) -> str:
                 return str(url.with_query(None))
 
             jar = aiohttp.DummyCookieJar()
-            if timeout_sec is not None:
-                timeout = aiohttp.ClientTimeout(total=timeout_sec)
-            else:
-                DEFAULT_TIMEOUT = aiohttp.ClientTimeout(total=5 * 60)
-                timeout = DEFAULT_TIMEOUT
+            # uses only global timeout and not runner specific timeouts
+            timeout_from_config = BentoMLContainer.config.runners.timeout.get()
+            timeout = aiohttp.ClientTimeout(total=timeout_from_config)
```

Comment:
lets use `@inject` and `Provide` here.

Reply:
@bojiang and I were discussing this and he suggested we might be better off not using provide here since this section is hot path and we can save a few lines since this is the only place where we need it the config.